benchmarks/lambda/analysis/lambda_plots.py

```diff
@@ -214,14 +215,16 @@ def gauss(x, A,mu, sigma):
     p0=(100, 0, 0.06)
     #print(bc[slc],y[slc])
     sigma=np.sqrt(y[slc])+(y[slc]==0)
-    coeff, var_matrix = curve_fit(fnc, list(bc[slc]), list(y[slc]), p0=p0,sigma=list(sigma))
-                                     
-    x=np.linspace(-1, 1)
-    sigmas.append(coeff[2])
-    dsigmas.append(np.sqrt(var_matrix[2][2]))
+    try:
+        coeff, var_matrix = curve_fit(fnc, list(bc[slc]), list(y[slc]), p0=p0,sigma=list(sigma))
```

Comment:
```suggestion
        coeff, var_matrix = curve_fit(fnc, list(bc[slc]), list(y[slc]), p0=p0,sigma=list(sigma), maxfev=100000)
```

Reply:
I just noticed this comment after merging.  I'll make similar edits to this and other curve_fit instances.